web/src/views/FindSpeaker.vue

```diff
@@ -34,6 +49,21 @@
             class="mb-5"
           />
         </div>
+        <v-row v-if="speakers.length">
```

Comment:
Let's DRY this up and make it into its own component. If you'd rather do this as a follow up PR so we can get the change into prod, that's okay too, just make a comment to let me know.

Reply:
you mean a specific component for speaker pagination? Yeah I can add it in a separate PR together with improved spacing of the arrows :)

Reply:
Great, I added a ticket #180 for tracking it.